src/main/webapp/i18n/en/complaint.json

```diff
@@ -8,7 +8,7 @@
             "info": "The complaint limit refers to open and rejected complaints. If your complaint is accepted it will not count towards this total.",
             "description": "You may submit up to {{ maxComplaintNumber }} complaints in this course.",
             "descriptionExam": "You can submit one complaint for each manually assessed exercise in this exam.",
-            "descriptionExtended": "Every student is allowed to submit up to {{ maxComplaintNumber }} complaints in this course. You still have <strong style=color:red;>{{ allowedComplaints }}</strong> complaints left.",
+            "descriptionExtended": "Every student is allowed to complain once on an exercise. In total {{ maxComplaintNumber }} complaints are possible in this course. You still have <strong style=color:red;>{{ allowedComplaints }}</strong> complaints left.",
             "descriptionTeam": "Your team may submit up to {{ maxComplaintNumber }} complaints in this course.",
             "descriptionTeamExtended": "Every team is allowed to submit up to {{ maxComplaintNumber }} complaints in this course. Your team still has <strong style=color:red;>{{ allowedComplaints }}</strong> complaints left.",
```

Comment:
_"descriptionTeamExtended": "One student per team can complain once on an exercise. A total of up to {{ maxComplaintNumber }} complaints per team can be submitted in this course. [...]_